libs/blocks/table/table.js

```diff
@@ -78,6 +78,27 @@ function handleHeading(table, headingCols) {
       col.innerHTML = '';
       col.append(row1, row2);
     }
+    const trackingHeader = col.querySelector('.tracking-header');
+    const nodeToApplyRoleScope = trackingHeader ?? col;
+
+    if (trackingHeader) {
+      const trackingHeaderID = `t${tableIndex + 1}-c${i + 1}-header`;
+      trackingHeader.setAttribute('id', trackingHeaderID);
+
+      const headerBody = col.querySelector('.body:not(.action-area)');
+      if (headerBody) headerBody.setAttribute('id', `${trackingHeaderID}-body`);
```

Comment:
We can simplify this too to avoid the `if` check by `headerBody?.setAttribute()`